engine/engine/src/engine_service.h

```diff
@@ -59,6 +59,13 @@ namespace dmEngineService
         dmResource::HFactory      m_Factory;
         dmGameObject::HRegister   m_Regist;
     };
+
+    struct EngineState
+    {
+        bool   m_ConnectionAppMode;
```

Comment:
for now we need only one variable, but in the future we are able to add more when we need it